tests/integration/test_hint.py

```diff
@@ -71,14 +72,45 @@ def test_cff(cff, tmpdir):
                    str(tmpdir / basename(out)) + ".xml"])
 
 
-@parametrize("path", glob.glob("%s/dummy/font.p*" % DATA_DIR))
-def test_type1(path, tmpdir):
+tx_found = False
+try:
+    subprocess.call(["tx", "-v"])
+    tx_found = True
+except OSError:
+    pass
+
+
+@parametrize("path", glob.glob("%s/dummy/font.pf[ab]" % DATA_DIR))
+@pytest.mark.skipif(tx_found, reason="'tx' is found")
+def test_type1_raises(path, tmpdir):
+    out = str(tmpdir / basename(path)) + ".out"
+    options = Options(path, out)
+    with pytest.raises(NotImplementedError):
+        hintFiles(options)
+
+
+@pytest.mark.parametrize("path", glob.glob("%s/dummy/font.ps" % DATA_DIR))
+def test_type1_cid_raises(path, tmpdir):
     out = str(tmpdir / basename(path)) + ".out"
     options = Options(path, out)
     with pytest.raises(NotImplementedError):
         hintFiles(options)
 
 
+@parametrize("path", glob.glob("%s/dummy/font.pf[ab]" % DATA_DIR))
+@pytest.mark.skipif(tx_found is False, reason="'tx' is missing")
+def test_type1_supported(path, tmpdir):
```

Comment:
What is this test actually doing? I'm confused by the two calls to `tx` below; I'd expect to be one call to tx, which would be diff'ed against expected results.

Reply:
It was supposed to dump input and output files and compares them, but now I see that there was a typo in the second call and both ended up dumping the input font, and I was wondering how the tests passed from the first time, LOL.

Reply:
I don't know. What exactly are the input and output files that the test is running on?

Reply:
The input is the Type 1 fonts under `tests/integration/data/dummy/`; `font.pfa`, `font.pfb` and `font.ps`. The output is what psautohint generates, which here will be roundtripped t1 → cff → t1 by tx, and I think that is the source of the difference.

Reply:
OK. The differences are gone now, after you round-tripped the input fonts with tx, yes?

Reply:
That test font is not that good, since it only contains one `FDArray` (it has been like that since it was added to the repo). Let me craft a better test font for that format.

Reply:
Added new **font.ps**: https://github.com/adobe-type-tools/psautohint-testdata/commit/5febaa3f9c98cbd4bf5c1da0a078883600cb7987